src/components/Card.js

```diff
@@ -12,42 +12,37 @@ export default function Creepypasta({
   isBookmarked,
 }) {
   return (
-    <Grid>
-      <Wrapper>
-        <BookmarkButton
-          onBookmarkClick={onBookmarkClick}
-          isBookmarked={isBookmarked}
-        />
-        <Card
-          style={{
-            backgroundImage: `url(${image})`,
-            backgroundSize: 'cover',
-            backgroundRepeat: 'no-repeat',
-          }}
-        >
-          <CardContent>
-            <Title>{title}</Title>
-            <Wordcount>{wordcount} Words</Wordcount>
-          </CardContent>
-          <ReadMorePosition>
-            <RedButton>
-              <ReadMoreButtonText to="/fullcreepypasta" state={{ id: id }}>
-                Read
-              </ReadMoreButtonText>
-            </RedButton>
-          </ReadMorePosition>
-        </Card>
-      </Wrapper>
-    </Grid>
+    <Wrapper>
+      <BookmarkButton
+        onBookmarkClick={onBookmarkClick}
+        isBookmarked={isBookmarked}
+      />
+      <Card
```

Comment:
I'm not sure why you choose the name 'Card' here, because your whole component is the 'Card' component. A different name would be better to understand. Maybe just `Image` is fine here. 🤔